src/coffee/directives/api/models/parent/windows-parent-model.coffee

```diff
@@ -223,5 +221,18 @@ angular.module('uiGmapgoogle-maps.directives.api.models.parent')
             interpModel[key] = model[key] for key in @contentKeys
             exp(interpModel)
 
+          modelKeyComparison: (model1, model2) =>
+            #handle possible transclusion
```

Comment:
complex override for windows transclusion to consider markerScope aka parentScope